Data/Transactions.txt

```diff
@@ -1,32 +1,8 @@
-- 
-Apsipirkimas iki
-123,5
-
--
-remontas
-999
-
-- 
-bariukas
-10
-
-+ 
-Skola
-25
-
--
-Parduotuve
-10
-
--
-aaa
-123
-
-+
-aaaa
-1234
-
--
-AAAA
-999
-
+{"TransactionType":"-","Amount":123.0,"Title":"asdsad","Category":"Living"}
```

Comment:
the structure looks pretty familiar to a JSON file. Maybe worth to use that instead of .txt?